database/models/fans.js

```diff
@@ -0,0 +1,134 @@
+const client = require('../index');
+
+module.exports = {
+  get: function(req, res) {
+    client.query(`
+      SELECT json_build_object(
+       'username', (SELECT username FROM authentication a WHERE a.id = f.authentication_id),
```

Comment:
'authentication' table named as 'auth' in database,
'authentication_id' named as 'auth_id' in fans table